ui/app/AppLayouts/Wallet/services/dapps/types/SessionRequestResolved.qml

```diff
@@ -31,9 +31,9 @@ QObject {
     // Data prepared for display in a human readable format
     required property var preparedData
 
-    readonly property alias dappName: d.dappName
-    readonly property alias dappUrl: d.dappUrl
-    readonly property alias dappIcon: d.dappIcon
+    property alias dappName: d.dappName
```

Comment:
Why do these props need to be aliases of props under the `d` object?
Guess we can remove entirely `id: d` object, and use them here, since also `property bool hasExpiry` is not used anywhere here.

Reply:
Agree! I have a pending work updating this. It's going to be included in my next PR.